test/pde_systems/MOL_1D_Linear_Diffusion_NonUniform.jl

```diff
@@ -3,6 +3,7 @@
 # Packages and inclusions
 using ModelingToolkit, MethodOfLines, LinearAlgebra, Test, OrdinaryDiffEq, DomainSets
 using ModelingToolkit: Differential
+using Random
```

Comment:
For testing purposes it is probably better to use the `StableRNG.jl` package with a fixed seed.